functions/src/Services/User.ts

```diff
@@ -1,10 +1,39 @@
 import { User, userConverter } from '../Models/User';
-import { db } from '../index';
+import { firestore } from 'firebase-admin';
```

Comment:
why are you import firestore db is firestore. you are basic doing more work also db is the firestore we initialized with the app. So i think you will need to change it back.